NuoDb.Data.Client/NuoDbDataReader.cs

```diff
@@ -76,7 +76,7 @@ private void InitResultSet(int handle, EncodedDataStream dataStream, bool readCo
             this.values = new Value[numberColumns];
             this.closed = false;
             this.currentRow = 0;
-            this.afterLast = false;
+            // this.afterLast = false;
```

Comment:
Why leave code commented out?

Reply:
This line was in a block that previously set ALL the member fields.

I agree that leaving a commented line is poor style.
I'll replace the commented line with a comment that explains why afterLast is not being set now.

The alternative was to set afterLast = false here, as originally done, and then set its value later.
But, I thought was was potentially confusing.